src/TemplateNPC.cpp

```diff
@@ -700,7 +700,7 @@ void sTemplateNPC::ExtractTalentTemplateToDB(Player* player, std::string& player
 void sTemplateNPC::ExtractGlyphsTemplateToDB(Player* player, std::string& playerSpecStr)
 {
     QueryResult result = CharacterDatabase.PQuery("SELECT glyph1, glyph2, glyph3, glyph4, glyph5, glyph6 "
-        "FROM character_glyphs WHERE guid = '%u' AND talentGroup = '%u';", player->GetGUID(), player->GetActiveSpec());
+        "FROM character_glyphs WHERE guid = '%u' AND talentGroup = '%u';", player->GetGUID().ToString().c_str(), player->GetActiveSpec());
```

Comment:
`GetCounter` instead of string.

Reply:
I was thinking about GetCounter, but I was not sure :sweat_smile: 
thanks!